Index spi (#162)
pinot-contrib/pinot-docs: basics/indexing/timestamp-index.md

Timestamp index is configured per column basis inside the fieldConfigList section in table config.

> Users need to define the field `timestampConfig` and specify the granularities that you want to index.

Reviewer: ```suggestion
Define the field `timestampConfig` and specify from the following granularities to index:
```
